src/routes/registry.ts

```diff
@@ -194,7 +194,7 @@ const resolveAccess = async (
 const authorizeRequest = (
 	req: Request,
 	scopes: string[],
-): Resolvable<Access[]> => {
+): Access[] | PromiseLike<Access[]> => {
```

Comment:
This is a weird bug where downstream a `Promise.all` type was suddenly adding `undefined` as a possible value